DiamondLightSource/python-artemis
# 301 zocalo interaction in handler class

Fixes #301 
Replaces #342 

Moves zocalo interaction code into the ZocaloHandlerCallback class; Adds a parameter to parameters and in order to be able to specify the data processing (zocalo) environment at runtime. ZocaloHandlerCallback defaults to production `artemis` environment while default parameters override this to `devrmq` for testing and so we don't accidentally spam prod zocalo.

### To test:
1. Inspect the changes and see if they are better than in #342 
2. Confirm tests pass
